docker image build failed

* Orion version:
* Python version: 
* Operating System: ubuntu 20.04

### Description
Hello, thanks to your library, I've been able to create a great program.

I'm trying to build a Docker image as described in DOCKER.md, but I'm encountering the following error in the terminal:

```

docker build -f docker/orion-jupyter.Dockerfile -t orion-jupyter .
[+] Building 1.5s (10/13)                                                                 docker:default
 => [internal] load .dockerignore                                                                   0.0s
 => => transferring context: 2B                                                                     0.0s
 => [internal] load build definition from orion-jupyter.Dockerfile                                  0.0s
 => => transferring dockerfile: 884B                                                                0.0s
 => [internal] load metadata for docker.io/library/python:3.6                                       1.4s
 => [1/9] FROM docker.io/library/python:3.6@sha256:f8652afaf88c25f0d22354d547d892591067aa4026a7fa9  0.0s
 => [internal] load build context                                                                   0.0s
 => => transferring context: 15.84kB                                                                0.0s
 => CACHED [2/9] RUN mkdir /app                                                                     0.0s
 => CANCELED [3/9] COPY setup.py /app                                                               0.0s
 => CACHED [4/9] RUN pip install -e /app && pip install jupyter                                     0.0s
 => CACHED [5/9] COPY orion /app/orion                                                              0.0s
 => ERROR [6/9] COPY notebooks /app/notebooks                                                       0.0s
------
 > [6/9] COPY notebooks /app/notebooks:
------
orion-jupyter.Dockerfile:10
--------------------
   8 |
   9 |     COPY orion /app/orion
  10 | >>> COPY notebooks /app/notebooks
  11 |     RUN mkdir -p /app/orion/notebooks/data \
  12 |         && cd /app/orion/notebooks/data \
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref 5a708955-7041-46b5-93f5-dbbb66f1a07f::n9okl95s634xfjhlo2e5xxx4l: "/notebooks": not found
make: *** [Makefile:277: docker-jupyter-build] Error 1
```

I suspect it might be because the 'notebooks' directory is missing. What should I do to resolve this?


### What I Did

```bash
make docker-jupyter-build
```

--- a/docker/orion-jupyter.Dockerfile
+++ b/docker/orion-jupyter.Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3.6
+FROM python:3.7
 
 EXPOSE 8888
 
@@ -7,17 +7,17 @@ COPY setup.py /app
 RUN pip install -e /app && pip install jupyter
 
 COPY orion /app/orion
-COPY notebooks /app/notebooks
-RUN mkdir -p /app/orion/notebooks/data \
-    && cd /app/orion/notebooks/data \
-    && wget https://d3-ai-orion.s3.amazonaws.com/S-1.csv \
-    && wget https://d3-ai-orion.s3.amazonaws.com/S-2.csv \
-    && wget https://d3-ai-orion.s3.amazonaws.com/E-1.csv \
-    && wget https://d3-ai-orion.s3.amazonaws.com/P-1.csv \
+COPY tutorials /app/tutorials
+RUN mkdir -p /app/orion/tutorials/data \
+    && cd /app/orion/tutorials/data \
+    && wget https://sintel-orion.s3.amazonaws.com/S-1.csv \
+    && wget https://sintel-orion.s3.amazonaws.com/S-2.csv \
+    && wget https://sintel-orion.s3.amazonaws.com/E-1.csv \
+    && wget https://sintel-orion.s3.amazonaws.com/P-1.csv \
     && mkdir -p /app/orion/data \
     && cd /app/orion/data \
-    && wget https://d3-ai-orion.s3.amazonaws.com/anomalies.csv \
-    && cp /app/orion/notebooks/data/* .
+    && wget https://sintel-orion.s3.amazonaws.com/anomalies.csv \
+    && cp /app/orion/tutorials/data/* .
 
 RUN adduser jupyter --uid 1000 --disabled-password --system && chown -R jupyter /app
 

--- a/docker/orion-jupyter.Dockerfile
+++ b/docker/orion-jupyter.Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3.7
+FROM python:3.6
 
 EXPOSE 8888
 

--- a/docker/package.sh
+++ b/docker/package.sh
@@ -8,20 +8,20 @@ mv orion-jupyter.tar Orion
 
 # Download the demo data inside the new repository
 cd Orion
-wget https://d3-ai-orion.s3.amazonaws.com/B-1.csv
+wget https://sintel-orion.s3.amazonaws.com/B-1.csv
 
-mkdir -p notebooks/data
-cd notebooks/data
-wget https://d3-ai-orion.s3.amazonaws.com/S-1.csv
-wget https://d3-ai-orion.s3.amazonaws.com/S-2.csv
-wget https://d3-ai-orion.s3.amazonaws.com/E-1.csv
-wget https://d3-ai-orion.s3.amazonaws.com/P-1.csv
+mkdir -p tutorials/data
+cd tutorials/data
+wget https://sintel-orion.s3.amazonaws.com/S-1.csv
+wget https://sintel-orion.s3.amazonaws.com/S-2.csv
+wget https://sintel-orion.s3.amazonaws.com/E-1.csv
+wget https://sintel-orion.s3.amazonaws.com/P-1.csv
 cd ../..
 
 mkdir -p orion/data
-cp notebooks/data/* orion/data
+cp tutorials/data/* orion/data
 cd orion/data
-wget https://d3-ai-orion.s3.amazonaws.com/anomalies.csv
+wget https://sintel-orion.s3.amazonaws.com/anomalies.csv
 
 cd ../../../
 

